packages/text-input-react/src/SupportText.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+interface Props {
+    helpText?: string;
+    errorText?: string;
+    isInvalid: boolean;
+}
+
+export const SupportText = ({ helpText, errorText, isInvalid }: Props) => {
```

Comment:
If we've made this into a component, should it be its own package in the library?

Reply:
Yeah, but I think that should be refactored when another component wants to use it